test/bidi.spec.mjs

```diff
@@ -98,12 +98,12 @@ describe('bidi.js', function() {
         });
     });
 
-    describe('thai scripts', () => {
+    describe('thai scripts', function() {
 
         let thaiFont;
         let bidiThai;
 
-        before(()=> {
+        beforeEach(()=> {
             thaiFont = loadSync('./test/fonts/NotoSansThai-Medium-Testing-v1.ttf');
```

Comment:
Why do we need to load/parse the font before each test? If it's not being modified and therefore interfering with following tests, this just adds overhead and increases testing runtime.

Reply:
it allows to use the same codebase on the 3 most popular test runner
yet it has no significant impact on the overall test execution time 

https://mochajs.org/#hooks use `.before()`/`.after()` :cry: 
https://bun.sh/docs/test/lifecycle use `.beforeAll()`/`.afterAll()`
https://jasmine.github.io/api/edge/global use `.beforeAll()`/`.afterAll()`

I could also switch from mocha to `bun test` (and save even more NPM dependencies)
```
npx mocha
npx bun test
npx jasmine "**/*.spec.mjs"
```

Or just create a wrapper in `test/testutil.mjs` with `before = before||beforeAll`